src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -720,26 +748,20 @@ public boolean configure(
                 StaplerRequest req,
                 JSONObject formData) throws FormException {
 
-            // to persist global configuration information,
-            // set that to properties and call save().
-            stashRootUrl = formData.getString("stashRootUrl");
-            ignoreUnverifiedSsl = formData.getBoolean("ignoreUnverifiedSsl");
-            includeBuildNumberInKey = formData.getBoolean("includeBuildNumberInKey");
+            this.stashRootUrl = null;
```

Comment:
I suggest to keep the order consistent for easier reading/comparison:
* field declarations in `StashNotifier` (maybe)
* setters/getters in `StashNotifier` (maybe)
* default values (lines 751-758) in `StashNotifier::configure`
* assertThat calls (lines 27-33) in `ConfigAsCodeTest::should_support_jcasc_from_yaml`
* setter calls (lines 40-47) in `ConfigAsCodeTest::should_support_jcasc_to_yaml`